webdriver-spec.html

```diff
@@ -2410,7 +2415,8 @@
 }</pre>
 </aside>
 
-<p>The <a>remote end steps</a> are:
+<p>The <a>remote end steps</a> <a data-lt="RES-arguments">with arguments</a> <var>url variables</var>
+ and <var>parameters</var> are:
```

Comment:
I think we can afford to be terse and not modify these lines. If the processing model says that the remote end steps accept _URL variables_ and _parameters_, that should be sufficient. Thoughts?